go.mod

```diff
@@ -1,9 +1,10 @@
 module github.com/stolostron/submariner-addon
 
 go 1.22.7
+toolchain go1.23.6
```

Comment:
@skitt Could this break anything downstream?

Reply:
Yes, downstream builders are currently on 1.23.4.